content/docs/04-clusters/01-public-cloud/03-azure/02-create-azure-cluster.md

```diff
@@ -122,6 +126,41 @@ You can add new worker pools to customize certain worker nodes to run specialize
 
 </InfoBox>
 
+## Azure Storage Account and Storage Container
+
+While deploying the Azure cluster, Palette creates a storage account and copies the base VHD image to the Palette default storage container within this storage account. This storage account will have unrestricted access and has an auto-generated name. But for specific use case scenarios, you can attach a custom storage account and storage containers to the Azure cluster. This section will emphasis on such use cases.
+
+### Use Cases
+
+You want to have a custom storage account to be used for the following reasons:
+* The use of a storage account with a custom name
+* A storage account with restricted access
+
+During Azure cluster provisioning, Palette allows linking custom storage accounts and containers to satisfy the above use case. This allows you to link the already created storage account from the UI drop-down option. 
+
+### Cases

```

Comment:
I'm not sure having another heading here is needed. We can include under Use Cases so that it's all together.